### ISSUE TITLE
`msDesc`/`additional` should always display last

### ISSUE THREAD

**Issue by holfordm** [bodleian/medieval-mss#205]
The `additional` element, as a child of `msDesc`, contains bibliographical and administrative information relating to the record as a whole, including any `msPart` elements. The default display of this information in document order, before any `msPart` elements, is potentially confusing. Displaying this information at the end of the record will provide an improved user experience and have the incidental benefit of making it easier to use `mspart` to describe endleaves.

**Comment by andrew-morrison** [comment]
I've made a change so that:

 * Anything within `msDesc/additional` is displayed after the last `msPart`.
 * If there are `msPart` elements, a subheading of "Additional Information" is added, in the same size font as the subheadings for each part, to indicate that what follows is not about the last part.
 * `msPart/additional` are left as-is.

I've re-indexed Medieval QA to provide some examples:

Old | New | Notes
------------ | ------------- | -------------
[MS. Canon. Liturg. 205](https://medieval.bodleian.ox.ac.uk/catalog/manuscript_2895) | [MS. Canon. Liturg. 205](http://medieval-qa.bodleian.ox.ac.uk/catalog/manuscript_2895) | Simple, no parts, therefore no difference
[Jesus College MS. 29](https://medieval.bodleian.ox.ac.uk/catalog/manuscript_10575) | [Jesus College MS. 29](http://medieval-qa.bodleian.ox.ac.uk/catalog/manuscript_10575) | Simple, short, 2 parts
[MS. Ashmole 244](https://medieval.bodleian.ox.ac.uk/catalog/manuscript_316) | [MS. Ashmole 244](http://medieval-qa.bodleian.ox.ac.uk/catalog/manuscript_316) | Longer, 8 parts
[MS. Digby 76](https://medieval.bodleian.ox.ac.uk/catalog/manuscript_4415) | [MS. Digby 76](http://medieval-qa.bodleian.ox.ac.uk/catalog/manuscript_4415) | 3 parts, the last of which has its own additional
[MS. Barocci 156](https://medieval.bodleian.ox.ac.uk/catalog/manuscript_984) | [MS. Barocci 156](http://medieval-qa.bodleian.ox.ac.uk/catalog/manuscript_984) | Flyleaves

As `additional` is always last except when there are `msPart`, testing in other catalogues produces little or no difference. In Fihrist, only 24 records are affected. So if this looks OK for Medieval, I'll send Mohammad an example (I can't re-index the production Fihirst web site currently, because they've got duplicate manuscript IDs.)

**Comment by holfordm** [comment]
This all looks good for medieval

**Comment by andrew-morrison** [comment]
Just a thought: Would it be preferable for the "Digital Images" section, i.e. `surrogates`, to be pulled out of the rest of `additional` and displayed at the top (say after and SC or other atlIdentifiers)?

**Comment by holfordm** [comment]
I don't think so, but it does raise the question of navigating long and complex entries 9which I don't really have an answer to: I don't really like how the Hebrew catalogue handles this (e.g. https://hebrew.bodleian.ox.ac.uk/catalog/volume_14) but maybe a stripped down version of that with just the part numbers?

**Comment by andrew-morrison** [comment]
There are always edge cases. Like MS. Lat. misc. a. 3, which has 87 parts. Or MS. Bodl. 572 with parts within parts containing works with sub-works.

One thing that could handle any size or complexity of description, without taking up a lot of space and cluttering the layout, would a drop-down, maybe in the top-right corner. It would jump to whatever part (or other major section?) you choose, using JavaScript. Here's a mockup:

![tei_catalogues_section_jumper_mockup](https://user-images.githubusercontent.com/33721187/56045473-3843d780-5d39-11e9-9134-5e14dc0386be.gif)

But I'm not sure how usable or accessible that would be.

**Comment by holfordm** [comment]
Andrew - would you be able to implement this drop down on QA so we can test it? We've got an increasing number of longer and more complex records, and I think it could be really helpful in those cases. Thanks!
